Scripts/Python/xCheat.py

```diff
@@ -678,6 +678,7 @@ def ImportMarkers(args):
                     if jnode.folderGetName() == mg[1]:
                         # yes, add the markers to this game
                         for marker in mg[0][4]:
+                            # TODO: PlasmaVaultConstants.PtVaultNodePermissionFlags does not exist
```

Comment:
This code is actually dead because markers don't work this way anymore. I'd remove this given there are bigger problems afoot 😉

Reply:
@Hoikas How much code did you want me to remove there? Or did you just mean remove my comment?

Reply:
Just the comment - fixing the actual function is definitely a "later" task.

Reply:
TODO Comment has been removed from this code